packages/automated-dispute/src/providers/protocolProvider.ts

```diff
@@ -1219,4 +1233,25 @@ export class ProtocolProvider implements IProtocolProvider {
 
         return bondEscalation;
     }
+
+    /**
+     * Checks if an operator is authorized for a given service provider and verifier.
+     *
+     * @param serviceProvider - The service provider address.
+     * @param verifier - The verifier address.
+     * @param operator - The operator address.
+     * @returns A promise that resolves to a boolean indicating authorization status.
+     */
+    async isAuthorized(
+        serviceProvider: Address,
+        verifier: Address,
+        operator: Address,
+    ): Promise<boolean> {
+        const authorized = await this.horizonStakingContract.read.isAuthorized([
+            serviceProvider,
+            verifier,
+            operator,
+        ]);
+        return authorized;
```

Comment:
```suggestion
        return this.horizonStakingContract.read.isAuthorized([
            serviceProvider,
            verifier,
            operator,
        ]);
```
picky here 😆 but given that you only return the result without logging or try/catch block